### ISSUE TITLE
user.set password module arguments missing

### ISSUE THREAD

**Issue by jeffclay** [saltstack/salt#19271]
The user.setpassword (aka win_useradd.setpassword) module works, but the module does not accept the extra arguments as described in http://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.win_useradd.html#salt.modules.win_useradd.setpassword .  The error given for the commands tried is listed below:

```
salt foo-1 user.setpassword user1 mypass neverexpires=True
foo-1:
    ERROR executing 'user.setpassword': The following keyword arguments are not valid: {'neverexpires': True}

salt foo-1 user.setpassword user1 mypass neverexpires:True
foo-1:
    TypeError encountered executing user.setpassword: setpassword() takes exactly 2 arguments (3 given). See debug log for more info.
```

**Comment by jfindlay** [comment]
@jeffclay, thanks for reporting this issue.

**Comment by jeffclay** [comment]
this seems to have been fixed in the dev version https://github.com/saltstack/salt/blob/develop/salt/modules/win_useradd.py   can this be updated or is 2014.7.x not getting updates due to future release of 2015.2.x ?

**Comment by babilen** [comment]
This feature was added in 714145d50b83889352d134abc0631e55f73825fb which is only in develop as of now. It therefore simply won't work on 2014.7.*.

**Comment by jfindlay** [comment]
That commit (#18840) is also available on the 2015.2 branch.  We are getting very close to releasing 2015.2.0 and I am wary of backporting so much code to an old stable branch.  @jeffclay, when 2015.2.0 comes out will you be able to verify if this is fixed?  Thanks.

**Comment by jeffclay** [comment]
yes, once the minion is released and the version is available in epel 7 repo.

**Comment by rallytime** [comment]
@jeffclay 2015.5.0 (we renamed the 2015.2 branch to better reflect the actual release date) was release a little while ago. Please give it a try at your earliest convenience and let us know if this is cleared up for you or not. Thanks again!

**Comment by twangboy** [comment]
Fixed on 2015.5 branch with #18840
